.circleci/config.yml

```diff
@@ -47,9 +47,34 @@ jobs:
       - run: npm run dist
       - run: bash <(curl -s https://codecov.io/bash) -F frontend
 
+  nightly-build:
+    docker:
+      - image: circleci/python:3.7.3
+    steps:
+      - checkout
+      - run: cd ~ && wget https://nodejs.org/dist/v8.7.0/node-v8.7.0-linux-x64.tar.gz
+      - run: cd /usr/local/ && sudo tar --strip-components 1 -xzf ~/node-v8.7.0-linux-x64.tar.gz
+      - run: pip install wheel boto3
+      - run: npm install
+      - run: npm run dist
+      - run: PYTHONPATH=. python scripts/nightly/get_version.py > wagtail/__init__.py
+      - run: python setup.py bdist_wheel
+      - run: python cripts/nightly/upload.py
```

Comment:
missing "s" in `python cripts/nightly/upload.py`. Should be `python scripts/nightly/upload.py`

Reply:
Oops, thanks!